src/implementations/config/create.ts

```diff
@@ -0,0 +1,211 @@
+import { CreateConfigCommand } from "../../commands/config/config.type";
+
+interface ConfigFile {
+  wallet: EncryptedWallet | AwsKmsWallet;
+  forms: Form[];
+}
+
+interface EncryptedWallet {
+  type: "ENCRYPTED_JSON";
+  encryptedJson: string;
+}
+
+interface AwsKmsWallet {
+  type: "AWS_KMS";
+  accessKeyId: string;
+  region: string;
+  kmsKeyId: string;
+}
+
+interface Form {
+  type: "VERIFIABLE_DOCUMENT" | "TRANSFERABLE_RECORD";
+  defaults: OpenAttestationDocument;
+}
+
+interface TypesOfForms {
+  type: "VERIFIABLE_DOCUMENT" | "TRANSFERABLE_RECORD";
+  identityProofTypes: (IdentityProofType | undefined)[];
+}
+
+const sandboxEndpointUrl = "https://sandbox.fyntech.io";
+
+export const create = async ({
+  encryptedWalletPath,
+  outputDir,
+  configType,
+  configTemplatePath,
+}: CreateConfigCommand): Promise<string> => {
+  let walletPath = "";
+  if (!encryptedWalletPath) {
+    info(`Wallet file not provided, please enter password to create a new wallet`);
+    const createWalletParams = {
+      outputFile: path.join(outputDir, "wallet.json"),
+      fund: "ropsten",
+    };
+    walletPath = await createWallet(createWalletParams);
+    info(`Wallet created at ${walletPath}`);
+  }
+  const walletFilePath = walletPath || encryptedWalletPath;
+  const wallet = await readFile(walletFilePath);
+  const walletObject = JSON.parse(wallet);
+
+  info(`Wallet detected at ${walletFilePath}`);
+
+  const configFile: ConfigFile = await selectTemplateFile(configType, configTemplatePath);
+  if (configFile.wallet.type === "ENCRYPTED_JSON") {
+    configFile.wallet.encryptedJson = wallet;
+  }
+
+  const formsInTemplate = configFile.forms;
+
+  const createDocumentStore = async (): Promise<string> => {
+    info(`Enter password to continue deployment of Document Store`);
+    const deployDocumentStoreParams = {
+      encryptedWalletPath: walletFilePath,
+      network: "ropsten",
+      gasPriceScale: 1,
+      dryRun: false,
+      storeName: "Document Store",
+    };
+    const { contractAddress } = await deployDocumentStore(deployDocumentStoreParams);
+    success(`Document store deployed, address: ${highlight(contractAddress)}`);
+    return contractAddress;
+  };
+
+  const createTokenRegistry = async (): Promise<string> => {
+    info(`Enter password to continue deployment of Token Registry`);
+    const deployTokenRegistryParams = {
+      registryName: "Demo Token Registry",
+      registrySymbol: "DTR",
+      encryptedWalletPath: walletFilePath,
+      network: "ropsten",
+      gasPriceScale: 1,
+      dryRun: false,
+    };
+    const { contractAddress } = await deployTokenRegistry(deployTokenRegistryParams);
+    success(`Token registry deployed, address: ${highlight(contractAddress)}`);
+    return contractAddress;
+  };
+
+  const generateDocumentStoreOrTokenRegistryOrDid = async (
+    typesOfForms: TypesOfForms[],
+    formType: "VERIFIABLE_DOCUMENT" | "TRANSFERABLE_RECORD",
+    didOrTxt: "DNS-DID" | "DNS-TXT"
```

Comment:
```suggestion
    identityProofType: "DNS-DID" | "DNS-TXT" | "DID"
```

Reply:
the "DID" we need? currently we dont have form using it thou